### ISSUE TITLE
Manifest version 2 is deprecated, and support will be removed in 2023. See https://developer.chrome.com/blog/mv2-transition/ for more details.

### ISSUE THREAD

**Issue by jomaro110** [sahilsehwag/pluralsight-downloader-chrome-extension#65]
The addon is not working anymore.

I have two errors:
1 - 
Manifest version 2 is deprecated, and support will be removed in 2023. See https://developer.chrome.com/blog/mv2-transition/ for more details.

2 -
Context
_generated_background_page.html
Stack Trace
_generated_background_page.html:0 (anonymous function)

**Comment by jomaro110** [comment]
And:
3 -
Error in event handler: TypeError: Cannot read properties of undefined (reading 'setBadgeBackgroundColor') at chrome-extension://khajmilinoeaobnggjgmjgffegfkimbp/src/background.js:43:24

4 -
Stack Trace
:0 (anonymous function)

**Comment by DWreck1995** [comment]
I can say that it does work for me on Chrome Version 117.0.5938.134

I did have to rename the CRX file to ZIP for Chrome to enable it. But other than that, everything works for me.

**Comment by dxps** [comment]
Indeed, I just managed to install it on my side on Chrome 117.0.5938.150 by:
1. Renaming `crx` to `zip`
    - Using `crx` didn't work: yet, the drag-n-drop installation worked, but I couldn't enable it due to the error that the extention is not from Chrome Web Store.
2. Using drag-n-drop into the Extensions page worked.
    - Using _Load unpacked_ fails, complaining about the manifest.

The error that is reported at the extension level (in _Extensions_ page, clicking _Errors_ on this extension tile) is shown below.
However, I can confirm it works! 🎉 

![image](https://github.com/sahilsehwag/pluralsight-downloader-chrome-extension/assets/14140226/2719ceb3-6d38-4120-8636-1cc4ccf24555)

**Comment by jomaro110** [comment]
It worked for me thank you.
Do you also need to download the videos one by one?
Before was when you download the playlist the download will start with the files automatically but now you need to save them one by one!

**Comment by DWreck1995** [comment]
I haven't had that issue. I start the course and hit E, then C, and it starts downloading.

However, in Chrome you have to make sure "ask where to save each file before downloading" is off. Otherwise it will prompt you each time it downloads a video. I also disabled "show downloads when they're done" because it will bring up the downloads pane each time and interrupt my multitasking.

![Downloads](https://github.com/sahilsehwag/pluralsight-downloader-chrome-extension/assets/81794049/eb6efefc-904a-4fc6-8841-f2027eb2e59e)

**Comment by jomaro110** [comment]
> I haven't had that issue. I start the course and hit E, then C, and it starts downloading.
> 
> However, in Chrome you have to make sure "ask where to save each file before downloading" is off. Otherwise it will prompt you each time it downloads a video. I also disabled "show downloads when they're done" because it will bring up the downloads pane each time and interrupt my multitasking.
> 
> ![Downloads](https://user-images.githubusercontent.com/81794049/273699936-eb6efefc-904a-4fc6-8841-f2027eb2e59e.png)

Thank you my friend, it's working great now 🥇

**Comment by obimelo** [comment]
I can't get the addon to work.

I have installed Chome 117.0.5938.150 and load extension in zip format, but can't download anything.

Can anyone check if it works?
